### PR TITLE
Fix crediting in slider picturator section of `osu!monthly, November 2022`

### PR DESCRIPTION
**Pull request by 0x84f**
As per [this comment](https://osu.ppy.sh/comments/2527686)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: news/2022-12-01-osumonthly-5.md [hunk 1]
```diff
@@ -89,9 +89,9 @@ If you are interested in watching the match yourself, you can find the [Twitch V
 
 ### OliBomby's Slider Picturator
 
-If you are at all interested in mapping on osu! you have probably already heard of [OliBomby](https://osu.ppy.sh/users/6573093) and his [Mapping Tools](https://mappingtools.github.io/) program that makes life easier for hundreds of mappers. But have you seen the latest update, or the [tweet](https://twitter.com/OliBomby/status/1592941194291863553?t=TEB1KmJ9pLdquaoVT9kv-g&s=19) from OliBomby on Twitter, revealing the Slider Picturator™?
+If you are at all interested in mapping on osu! you have probably already heard of [OliBomby](https://osu.ppy.sh/users/6573093) and his [Mapping Tools](https://mappingtools.github.io/) program that makes life easier for hundreds of mappers. But have you seen the latest update, or the [tweet](https://twitter.com/OliBomby/status/1592941194291863553?t=TEB1KmJ9pLdquaoVT9kv-g&s=19) from OliBomby on Twitter, revealing the Slider Picturator&trade;?
```
Comment (0x84f) [inline news/2022-12-01-osumonthly-5.md:92]:
```suggestion
If you are at all interested in mapping on osu! you have probably already heard of [OliBomby](https://osu.ppy.sh/users/6573093) and his [Mapping Tools](https://mappingtools.github.io/) program that makes life easier for hundreds of mappers. But have you seen the latest update, or the [tweet](https://twitter.com/OliBomby/status/1592941194291863553?t=TEB1KmJ9pLdquaoVT9kv-g&s=19) from OliBomby on Twitter, revealing the Slider Picturator™?
```

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — DISMISSED** [review]
works too

**Review by Coppertine — CHANGES_REQUESTED** [review]
I would modify the header to JPK314's Slider PicturatorTM

File: news/2022-12-01-osumonthly-5.md (hunk 1 above)
Comment (0x84f) [reply news/2022-12-01-osumonthly-5.md:92]:
```suggestion
If you are at all interested in mapping on osu! you have probably already heard of [OliBomby](https://osu.ppy.sh/users/6573093) and his [Mapping Tools](https://mappingtools.github.io/) program that makes life easier for hundreds of mappers. But have you seen the latest update, or the [tweet](https://twitter.com/OliBomby/status/1592941194291863553?t=TEB1KmJ9pLdquaoVT9kv-g&s=19) from OliBomby on Twitter, revealing the Slider Picturator™?
```

Comment (0x84f) [reply news/2022-12-01-osumonthly-5.md:92]:
```suggestion
If you are at all interested in mapping on osu! you have probably already heard of [OliBomby](https://osu.ppy.sh/users/6573093) and his [Mapping Tools](https://mappingtools.github.io/) program that makes life easier for hundreds of mappers. But have you seen the latest update, or the [tweet](https://twitter.com/OliBomby/status/1592941194291863553?t=TEB1KmJ9pLdquaoVT9kv-g&s=19) from OliBomby on Twitter, revealing the Slider Picturator™?
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**